### ISSUE TITLE
Filter Projects Page

### ISSUE THREAD

**Issue by George-CAPHPI** [davidroberts63/OctoPygmy#47]
I would love to have the same filter functionality from the dashboard screen applied to the full Projects list (i.e. filter by project group).

**Comment by davidroberts63** [comment]
I recall a few others mentioning this as well.

**Comment by bluetrack** [comment]
Please do consider this enhancement.  The concern/issue, as it stands, is that customizing the dashboard only shows explicit project additions.  If new projects are created under a project group those have to be added to the custom dashboard manually by each user.

**Comment by davidroberts63** [comment]
Yeah. I'd also like to try to add to the 'filtering on' button an ability to quickly switch back to a 'view all' projects. To help with the situation where you primarily work on a small set of projects but occasionally work on a different one, but don't want the hassel of re-configuring your dashboard.
